audio_service_web/pubspec.yaml

```diff
@@ -0,0 +1,37 @@
+name: audio_service_web
+description: Flutter plugin to play audio in the background while the screen is off.
```

Comment:
The description will need to change to mention "web". There is a pana penalty if the description is too short (e.g. web implementation of audio_service).